args_test.go

```diff
@@ -93,102 +93,280 @@ func rangeArgsWithInvalidCount(err error, t *testing.T) {
 	}
 }
 
+// NoArgs
+
 func TestNoArgs(t *testing.T) {
 	c := getCommand(NoArgs, false)
 	output, err := executeCommand(c)
 	expectSuccess(output, err, t)
 }
 
-func TestNoArgsWithArgs(t *testing.T) {
+func TestNoArgs_WithArgs(t *testing.T) {
 	c := getCommand(NoArgs, false)
-	_, err := executeCommand(c, "illegal")
+	_, err := executeCommand(c, "one")
+	noArgsWithArgs(err, t)
+}
+
+func TestNoArgs_WithValid_WithArgs(t *testing.T) {
+	c := getCommand(NoArgs, true)
+	_, err := executeCommand(c, "one")
 	noArgsWithArgs(err, t)
 }
 
+func TestNoArgs_WithValid_WithInvalidArgs(t *testing.T) {
+	c := getCommand(NoArgs, true)
+	_, err := executeCommand(c, "one")
```

Comment:
Based on the title of the test, the arg here should be "a".  Currently this is the exact same test as the previous one.

Reply:
Done in https://github.com/spf13/cobra/pull/1643/commits/6863083cddaccf6e875723c9facf3e2216a88882.